### ISSUE TITLE
Share price calculation bug

### ISSUE THREAD

**Issue by jennazenk** [melonproject/protocol#352]
Share price obtained from call to performCalculations seems to always return 1. 

After setup fund, prior to investing, call to performCalculations returns : 
````
Initial calculations- GAV: 0, NAV: 0, Share Price: 1, totalSupply: 0
````
Returned values are expected/correct. 

After investing 20 MLN in the previously created fund, call to performCalculations returns : 
````
Mid calculations- GAV: 20, NAV: 20, Share Price: 1, totalSupply: 20
````
Returned values are expected/correct. 

After making an order and taking 2 orders form the exchange, call to performCalculations returns : 
````
End calculations- GAV: 23.595905310300704 NAV: 23.595905310300704, Share Price: 1, totalSupply: 20
````
Returned value for sharePrice is incorrect.

After some trading activity, ie making and taking orders, share price is still 1 while GAV and NAV have evolved: as share price calculation depends on NAV/GAV, it is expected that when GAV/NAV fluctuate, share price shall fluctuate accordingly. In the above example, share price is expected to be ~1.18.

**Comment by jennazenk** [comment]
Seems fixed in protocol@0.6.2-deploy.9 ; no tag existing yet  since this deployment meant to be experimental. I think @travs will release a new version including the fix and with a git tag this afternoon. 

One problem left, pre flight calls to risk management :)

**Comment by jennazenk** [comment]
Solved here https://github.com/melonproject/protocol/pull/319/commits/b22eb6e5ed560224ac9d795fdf268580158096da
